mobile_verifier/src/boosting_oracles/assignment.rs

```diff
@@ -28,6 +35,19 @@ impl From<Assignment> for i32 {
     }
 }
 
+impl TryFrom<i32> for Assignment {
```

Comment:
I would prefer we not match on the i32 value but the enum value
```
impl From<oracle_boosting_hex_assignment::Assignment> for Assignment {
    fn from(value: oracle_boosting_hex_assignment::Assignment) -> Self {
        match value {
            oracle_boosting_hex_assignment::Assignment::A => Self::A,
            oracle_boosting_hex_assignment::Assignment::B => Self::B,
            oracle_boosting_hex_assignment::Assignment::C => Self::C,
        }
    }
}

```
Then we can use the built-in functions to the proto enum to get the enum value ex. `hex.urbanized()` rather than `hex.urbanized`

Reply:
Yes! That's much nicer, thank you.